components/conference/conferenceSchedule.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import { Container, Tab, Tabs } from 'react-bootstrap';
+
+import citySchedulesData from './../data/CitySchedulesData.json';
+import styles from './ConferenceSchedule.module.less';
+
+export const CityScheduleInfo: FC = () => (
+  <Container>
+    <section
+      className={`${styles.container} mx-auto my-0 position-relative ${styles.animated} text-center`}
+    >
+      <h2 id="schedule" className="fs-4 py-5">
+        大会日程与报名通道
+      </h2>
+
+      <Tabs
+        variant="pills"
+        className={`${styles.sm_max_width} w-75 mb-2`}
+        justify
+      >
+        {citySchedulesData.map(({ city, href, schedules }) => (
+          <Tab className="p-3" key={city} eventKey={city} title={city}>
+            <div
+              className={`${styles.table_box} w-100 overflow-x-auto`}
+              data-city={city}
+              key={city}
+            >
+              <a
+                target="_blank"
+                href={href}
+                rel="noreferrer"
+                className="px-3 pb-3 text-start d-block"
+              >
+                {href ? '点此报名' : '报名通道即将开启'}
+              </a>
+              {schedules !== undefined && schedules.length !== 0 ? (
+                <table className={`${styles.sm_max_width} text-start w-75`}>
+                  <thead>
+                    <tr>
+                      <th className="px-3 py-3 ">时间</th>
+                      <th className="px-3 py-3 ">主题</th>
+                      <th className="px-3 py-3">演讲嘉宾</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {schedules.map(({ time, topic, guest }) => (
+                      <tr key={topic}>
+                        <td className="px-3 py-2">{time}</td>
+                        <td className="py-2">{topic}</td>
+                        {guest ? <td className="py-2">{guest}</td> : <></>}
+                      </tr>
+                    ))}
+                  </tbody>
+                </table>
+              ) : (
+                <></>
```

Comment:
```suggestion
              {schedules?.[0] && (
                <table className={`${styles.sm_max_width} text-start w-75`}>
                  <thead>
                    <tr>
                      <th className="p-3">时间</th>
                      <th className="p-3">主题</th>
                      <th className="p-3">演讲嘉宾</th>
                    </tr>
                  </thead>
                  <tbody>
                    {schedules.map(({ time, topic, guest }) => (
                      <tr key={topic}>
                        <td className="px-3 py-2">{time}</td>
                        <td className="py-2">{topic}</td>
                        <td className="py-2">{guest}</td>
                      </tr>
                    ))}
                  </tbody>
                </table>
```
1. https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Optional_chaining
2. 表格 `<th />`、`<td />` 数目要对上

Reply:
> 如果 演讲嘉宾信息缺失， 也要对上吗 ？ 

没有那个单元格就是空的，但如果没有单元格就很丑。